react/src/pages/dashboard/SamplesTable.tsx

```diff
@@ -2,6 +2,8 @@ import React, { useState } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import Chip from '@material-ui/core/Chip';
 import MaterialTable, { MTableToolbar } from 'material-table';
+import { Cancel } from '@material-ui/icons';
+import { IconButton } from '@material-ui/core';
```

Comment:
Could you combine all the `core` imports as one statement since there's a few now? Bundle size should not be a concern since [material-ui supports tree-shaking](https://material-ui.com/guides/minimizing-bundle-size/).